### DISCUSSION TITLE
Regrouping cpv code with nature

### DISCUSSION THREAD

**Discussion by MiraRatajczak** [OP-TED/eForms-SDK#685] [Ideas]
Why is the cpv code separated from the nature and not in the same group? These are linked together with the business rule that checks if the Main classification correspond the nature of the contract.

**Comment by rouschr** [comment]
I am not a business person but Enrico said that these are separate nodes.
Main classification:
```
"xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cac:MainCommodityClassification/cbc:ItemClassificationCode/@listName",
```

Main nature:
```
"xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cbc:ProcurementTypeCode[@listName='contract-nature']",
```
and also CPV is not alone.

**Comment by bertrand-lorentz** [comment]
As this is an idea/suggestion, I'm converting it to a discussion.
